shared/src/userData.ts

```diff
@@ -11,8 +11,8 @@ export interface UserData {
   readonly lastUpdatedISO: string;
   readonly dateCreatedISO: string;
   readonly versionWhenCreated: number;
-  readonly businesses: Record<string, Business>;
-  readonly currentBusinessId: string;
+  businesses: Record<string, Business>;
+  currentBusinessId: string;
```

Comment:
Removed `readonly` from 'businesses' and 'currentBusinessId' properties to allow mutability. These properties need to be updated when syncing data (e.g., from the users table to the businesses table). Removing 'readonly' ensures that we can modify these properties during the sync process without causing TypeScript errors.